File: python/7 kyu/237.py
# https://www.codewars.com/kata/58f6f87a55d759439b000073/train/python

# Passed

def negation_value(s: str, val) -> bool:
    boolean_val = bool(val)
    for _ in s:
        boolean_val = not(boolean_val)
        
    return boolean_val
        
output = negation_value("!!!", [])
print(output)

File: nodejs/7 kyu/237.js
// https://www.codewars.com/kata/6965d769930fb2eff921668f/train/javascript

// Passed

function solution(first, second) {
  while (first >= second - first) {
    [second, first] = [first, second - first];
  }

  let result = [first, second];
  return result;
}

const output = solution(398, 644);
console.log(output);